parts/windows/windowscsehelper.ps1

```diff
@@ -26,6 +26,23 @@ function Postpone-RestartComputer
     Write-Log "Created an one-time task to restart the VM"
 }
 
+function Create-Directory
+{
+    Param(
+        [Parameter(Mandatory=$true)][string]
+        $Folder,
+        [Parameter(Mandatory=$false)][string]
+        $FolderUsage = "general purpose"
+    )
+    
+    if (-Not (Test-Path $Folder)) {
+        Write-Log "Create folder $Folder for $FolderUsage"
+        New-Item -ItemType Directory -Path $Folder > $null

```

Comment:
Will this also be able to make parent directories as needed, like `make -p`

Reply:
Just do the same function as old command `mkdir` and `New-Item -ItemType Directory` do. Do not introduce new functionality.

Reply:
`New-Item -ItemType Directory -Path $Folder > $null` is able to make parent directories as needed per my test.